salsa_dancing_molecules/simulations/nve.py

```diff
@@ -35,16 +38,26 @@ def run_for_materials(formula, api_key, steps, output_path, repeat=0):
         if repeat > 0:
             atoms = atoms.repeat(repeat)
 
+        if potential == '':
+            print('No potential was given, will use standard Lennard-Jones '
+                  'potential')
+
         try:
             print(f'Simulating for {symbols}')
-            run(atoms, steps, output_name)
+            run(atoms, steps, output_name, potential)
+        except ValueError as e:
+            print(f'Size error for {symbols}: {e}')
+            print('Increasing amount of cells')
+            repeat = repeat + 2
+            atoms = atoms.repeat(repeat)
+            run(atoms, steps, output_name, potential)
```

Comment:
what happens if this also does not work?

Reply:
Have changed it to a while loop that will continue until it is large enough.